src/modules/user/user.service.ts

```diff
@@ -0,0 +1,14 @@
+import { UserRepo } from './index';
+import { type IUser, type IUserInstance } from './type';
+
+class UserService {
```

Comment:
your implementation of UserService is too dummy. Instead, UserService should be responsible for all manipulation with user data. But you moved it to controller lvl, what is wrong